indico/web/client/styles/semantic_ui_overrides.scss

```diff
@@ -0,0 +1,12 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+// Semantic UI React adds a .hidden to checkboxes making them inaccessible to
+// keyboard users. This override restores the default presentation.
+.ui.checkbox > input {
```

Comment:
The `.hidden { display: none; }` is coming from our custom CSS, not from SUI... 

So maybe it's better to explicitly override it as `.ui.checkbox > input.hidden`?

Reply:
Ah, ok. I'd better close the upstream issue then. :)

And yeah, I think it's better to specifically target the `.hidden` class. I'll fix that and rebase on master in the process.

Reply:
Done.